tests/modules/images/webp-uploads/helper-tests.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Tests for webp-uploads module helper.php.
+ *
+ * @package performance-lab
+ * @group   webp-uploads
+ */
+
+use PerformanceLab\Tests\TestCase\ImagesTestCase;
+
+class WebP_Uploads_Helper_Tests extends ImagesTestCase {
```

Comment:
Same comment as the previous one.

```suggestion
class WebP_Uploads_Helper_Tests extends WP_UnitTestCase {
```